### DISCUSSION TITLE
doris-be CPU资源占用过高 是什么问题呢？

### DISCUSSION THREAD

**Discussion by cst4049** [apache/doris#26157] [A - General / Q&A]
doris版本：2.0.2
部署方式： 1fe, 2be
部署类型 k8s部署
机器 16C 64G

每秒进来数据量大概1w条

doris-be CPU两台资源占用都是800%

I1031 07:01:53.834678  2188 engine_publish_version_task.cpp:231] finish to publish version on transaction.transaction_id=591023, cost(us): 9112, error_tablet_size=0, res=[OK]
I1031 07:01:53.834733  2188 task_worker_pool.cpp:1613] successfully publish version|signature=591023|transaction_id=591023|tablets_num=45|cost(s)=0
I1031 07:01:53.851948  1814 routine_load_task_executor.cpp:257] finished routine load task id=b4fbd8aebec84a88-b7a048c0095ed1f7, job_id=33352, txn_id=591023, label=raw_log-33352-b4fbd8aebec84a88-b7a048c0095ed1f7-591023, elapse(s)=13, status: [OK], current tasks num: 1
I1031 07:01:53.882576  2348 load_channel.cpp:46] load channel removed. mem peak usage=14716926, info=label: LoadChannel#senderIp=doris-be-1.doris-be.myrz-9ceacc.svc.cluster.local#loadID=08819b7c9b104893-a956e2d7a79c3335; consumption: 0; peak_consumption: 14716926; , load_id=08819b7c9b104893-a956e2d7a79c3335, is high priority=1, sender_ip=doris-be-1.doris-be.myrz-9ceacc.svc.cluster.local
I1031 07:01:53.896574 343349 task_worker_pool.cpp:267] successfully submit task|type=PUBLISH_VERSION|signature=591019|queue_size=1
I1031 07:01:53.896826  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36051.1687037434.fd4f9d6821d0fce8-a2f470b7c08620ae, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 193(us) 
I1031 07:01:53.896876  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36057.1687037434.9842dc0ad0053f62-4fa99c3d6f85a989, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 237(us) 
I1031 07:01:53.896936  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36061.1687037434.7a46a24ab19d1694-6e06f7c6bb47c9ba, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 292(us) 
I1031 07:01:53.896957  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36065.1687037434.654119fc9b788b37-eaaa408b8adb5791, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 307(us) 
I1031 07:01:53.897027  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36067.1687037434.a549973b43501748-6f191ddaa133179f, transaction_id=591019, version=29147, num_rows=763, res=[OK], cost: 373(us) 
I1031 07:01:53.897037  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36069.1687037434.e14152e398c7a06a-a5e6eb0ba54918bb, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 377(us) 
I1031 07:01:53.897094  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36075.1687037434.484d2f3ed76c387a-e9e0900d9149f883, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 431(us) 
I1031 07:01:53.897123  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36077.1687037434.f444346ee222d503-7a4e33b56eab5bad, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 454(us) 
I1031 07:01:53.897156  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36089.1687037434.644cefc17dac516e-5c26b03a49aec792, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 481(us) 
I1031 07:01:53.897187  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36093.1687037434.a14eb8f5c39a8399-08e9e20e0837f5a4, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 507(us) 
I1031 07:01:53.897222  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36097.1687037434.29403b3da6ad806f-ebe88add4d05b3a1, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 536(us) 
I1031 07:01:53.897254  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36099.1687037434.9747c9dec375608b-41c527688bf1b79e, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 562(us) 
I1031 07:01:53.897291  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36101.1687037434.c442b85fbcb2326a-fb4813ab1c6787af, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 595(us) 
I1031 07:01:53.897320  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36105.1687037434.9348979c496bdb6f-a3cb4c0148a225be, transaction_id=591019, version=29147, num_rows=2519, res=[OK], cost: 617(us) 
I1031 07:01:53.897353  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36113.1687037434.e24719ed2c23bb7b-060a4b6f602dc1a5, transaction_id=591019, version=29147, num_rows=1201, res=[OK], cost: 647(us) 
I1031 07:01:53.897385  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36115.1687037434.3d49f829bafefebc-0bf3dfcd715a63bd, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 665(us) 
I1031 07:01:53.897414  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36117.1687037434.0c4f7cf0eba48656-3c34bbe17a9f4090, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 698(us) 
I1031 07:01:53.897444  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36121.1687037434.2f44f5ba34e72bca-496d48e3b540f2a4, transaction_id=591019, version=29147, num_rows=2604, res=[OK], cost: 723(us) 
I1031 07:01:53.897555  2140 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36127.1687037434.b04ec2b8d59ff3c8-737179a147fc19bb, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 823(us) 
I1031 07:01:53.897557  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36125.1687037434.a84bb3ec62e867c4-bbd9262fceeb4fbb, transaction_id=591019, version=29147, num_rows=2045, res=[OK], cost: 829(us) 
I1031 07:01:53.897644  2154 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36129.1687037434.2a489870fe8bf8d2-075a57f08853c3b3, transaction_id=591019, version=29147, num_rows=0, res=[OK], cost: 904(us) 
I1031 07:01:53.897650  2153 engine_publish_version_task.cpp:289] publish version successfully on tablet, table_id=34159, tablet=36131.1687037434.c94d5016a88b4491-d43f0e8fb0d47282, transaction_id=591019, version=29147, num_rows=2586, res=[OK], cost: 907(us) 
I1031 07:01:53.897682  2183 engine_publish_version_task.cpp:231] finish to publish version on transaction.transaction_id=591019, cost(us): 1098, error_tablet_size=0, res=[OK]
I1031 07:01:53.897749  2183 task_worker_pool.cpp:1602] trigger compaction succ, tabletid:36075, publised:10220
I1031 07:01:53.897783  2183 task_worker_pool.cpp:1602] trigger compaction succ, tabletid:36125, publised:8550
I1031 07:01:53.897791  2183 task_worker_pool.cpp:1613] successfully publish version|signature=591019|transaction_id=591019|tablets_num=22|cost(s)=0
I1031 07:01:53.899190  2090 compaction.cpp:199] start to do ordered data compaction, tablet=36075.1687037434.484d2f3ed76c387a-e9e0900d9149f883, output_version=[29138-29147]
I1031 07:01:53.901648  2085 compaction.cpp:324] start cumulative compaction. tablet=36125.1687037434.a84bb3ec62e867c4-bbd9262fceeb4fbb, output_version=[29118-29147], permits: 12
I1031 07:01:53.901698  2085 merger.cpp:341] Start to do vertical compaction, tablet_id: 36125
I1031 07:01:53.902127  2090 compaction.cpp:312] succeed to do ordered data cumulative compaction. tablet=36075.1687037434.484d2f3ed76c387a-e9e0900d9149f883, output_version=[29138-29147], disk=/opt/apache-doris/be/storage, segments=0, input_row_num=0, output_row_num=0. elapsed time=0.002982s. cumulative_compaction_policy=size_based
I1031 07:01:53.963258  2085 compaction.cpp:498] succeed to do cumulative compaction is_vertical=1. tablet=36125.1687037434.a84bb3ec62e867c4-bbd9262fceeb4fbb, output_version=[29118-29147], current_max_version=29147, disk=/opt/apache-doris/be/storage, segments=5, input_row_num=11789, output_row_num=11789. elapsed time=0.061853s. cumulative_compaction_policy=size_based, compact_row_per_second=190566

**Comment by LemonLiTree** [comment]
`top -H`
